float64 sample_time
string session_state
string connection_quality
string safety_state
string operation_mode
string drive_state
string client_command_mode
string overlay_type
string control_mode
float64 timestamp
float64[] measured_joint_position
float64[] commanded_joint_position
float64[] interpolator_joint_position
float64[] measured_joint_torque
float64[] commanded_joint_torque
float64[] external_joint_torque
float64 tracking_performance
